src/app/components/kudos-challenge-completed/kudos-challenge-completed.service.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+    "use strict";
+    angular.module("myApp.components.completedChallenges")
+        .factory("CompletedChallengesService", CompletedChallengesService);
+
+    CompletedChallengesService.$inject = [
+        "Challenges"
+    ];
+
+    function CompletedChallengesService(challengesBackend) {
```

Comment:
It's only my opinion - but is this service actually necessary? You're wrapping an `angular.factory` inside another `angular.factory` just for the sake of it. Usually, services have some sort of logical purpose, where they contain several service methods which belong to the same logical area and provide different parts of info for the same view. This service just pipes the return response of one  method from the backend- doesn't even return a promise or anything.  IMO, this could be combined with `KudosChallengeNewService` and, probably, something else to make a unified factory which deals with providing challenge info in a logical manner.